grails-app/utils/org/broadinstitute/orsp/utils/IssueUtils.groovy

```diff
@@ -105,25 +106,20 @@ final class IssueUtils {
         null
     }
 
-    static Map<String, Object> generateArgumentsForRedirect(Issue issue, String id, String tab) {
+    static Map<String, Object> generateArgumentsForRedirect(String type, String id, String tab) {
         Map<String, Object> arguments = new HashMap<>()
-        if (isConsentGroup(issue)) {
+        if (type == IssueType.CONSENT_GROUP.name) {
```

Comment:
I don't understand why this change (in combination with removing `isConsentGroup()`) was necessary. Are there issues with the wrong controller? Also, this change seems very out of scope for the task of improving search performance.

Reply:
It seems out of scope but I found out that it was simpler to use this method with a String for the IssueType than sending the Issue. And also, and because I couldn't use this method with IssueSearchItemDTO. While I was figuring out how to use this method with Issue Search DTO I found that this approach was simpler, solve my problem and needed less code.